app/build.gradle

```diff
@@ -97,22 +90,13 @@ dependencies {
     debugImplementation 'androidx.test:runner:1.3.0'
     debugImplementation 'androidx.test:rules:1.3.0'
 
-    // Espresso dependencies
+    // Espresso
     debugImplementation 'androidx.test.espresso:espresso-core:3.3.0'
-    androidTestImplementation 'androidx.test.espresso:espresso-contrib:3.3.0'
-    androidTestImplementation 'androidx.test.espresso:espresso-intents:3.3.0'
-    androidTestImplementation 'androidx.test.espresso:espresso-accessibility:3.3.0'
-    androidTestImplementation 'androidx.test.espresso:espresso-web:3.3.0'
+
     debugImplementation 'androidx.fragment:fragment-testing:1.2.5'
-    androidTestImplementation('com.schibsted.spain:barista:3.7.0') {
-        exclude group: 'org.jetbrains.kotlin'
-    }
+    androidTestImplementation('com.schibsted.spain:barista:3.7.0')
     androidTestImplementation 'androidx.test:core-ktx:1.3.0'
     androidTestImplementation 'androidx.test.ext:junit-ktx:1.1.2'
 
     kaptAndroidTest "com.google.dagger:dagger-compiler:$dagger_version"
```

Comment:
Same here, let's extract all the versions into the project level `build.gradle`.

Reply:
Done.